addons/website/static/src/components/dialog/seo.js

```diff
@@ -457,6 +457,9 @@ export class OptimizeSEODialog extends Component {
             if (seoContext.seoName !== this.previousSeoName) {
                 data.seo_name = seoContext.seoName;
             }
+            if (!seoContext.seoName) {
+                seoContext.seoName = this.seoNameDefault;
```

Comment:
As this guard clause is here to prevent redirection issues, why not putting it just before the redirection (`this.website.goToWebsite`...)?
It seems that the 'if canEdit' blocks are here to update the data to pass to the orm whenever it's necessary. We may keep them limited to that purpose.

Just a suggestion:
```
        this.website.goToWebsite({
            path: this.url.replace(
                this.previousSeoName || this.seoNameDefault,
                seoContext.seoName || this.seoNameDefault
            ),
        });
```

Reply:
Hello @lm4649, Thanks for pointing out, adapted the changes. Thanks :)